app/src/main/java/com/example/trainingintive/di/AppModule.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.trainingintive.di
+
+import android.app.Application
+import android.content.Context
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+
+@InstallIn(SingletonComponent::class)
+@Module
+class AppModule {
+
+    @Provides
+    fun provideApplicationContext(application: Application): Context = application.applicationContext
```

Comment:
Hilt provides application context of out the box :D Just use annotation `@ApplicationContext` where you need it